src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Models/SasToken.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+
+namespace Kaiyuanshe.OpenHackathon.Server.Models
+{
+    /// <summary>
+    /// Shared Access Signature Token. Reference: https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview
+    /// </summary>
+    public class SasToken
+    {
+        /// <summary>
+        /// Token for SAS
+        /// </summary>
+        /// <example>reFh8jikl%2B2hOVhJ5SORPkItoFN6Y8bbq%2FeYgKn6QsI%3D</example>
+        [Required]
+        [MinLength(1)]
+        [MaxLength(10240)]

```

Comment:
looks like `SasToken` is for response only? then `Required`, `MinLength` and `MaxLength` are not used. They are only for request validation

Reply:
yes. `SasToken` is for response only